app/frontend/src/features/communities/CommunityPage/EventCard.tsx

```diff
@@ -0,0 +1,99 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import classNames from "classnames";
+import React from "react";
+import LinesEllipsis from "react-lines-ellipsis";
+
+import { Card, CardActionArea, CardContent } from "../../../components/Card";
+import { CalendarIcon, ClockIcon } from "../../../components/Icons";
+import { timestamp2Date } from "../../../utils/date";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 120,
+  },
+  title: {
+    marginTop: 0,
+    marginBottom: 0,
+  },
+  subtitle: { marginBottom: theme.spacing(0.5) },
+  icon: {
+    display: "block",
+    marginInlineEnd: theme.spacing(0.5),
+    fontSize: "1rem",
+  },
+  detailsList: {
+    "ul&": {
+      listStyle: "none",
+      margin: 0,
+      padding: 0,
+    },
+    "& > li": {
+      display: "flex",
+      alignItems: "center",
+    },
+  },
+}));
+
+export default function EventCard({
+  event,
+  className,
+}: {
+  event: {
+    title: string;
+    creatorName: string;
+    location: string;
+    startTime: { seconds: number; nanos: number };
+  };
+  className?: string;
+}) {
+  const classes = useStyles();
+  const date = timestamp2Date(event.startTime);
+  return (
```

Comment:
Not sure if this is only in Storybook... but I feel this should show the whole name since the test name you got isn't even that long?
![Screenshot 2021-02-26 at 00 41 27](https://user-images.githubusercontent.com/13669362/109238438-729eac80-77cb-11eb-93ad-4b07a19f6300.png)

Reply:
Maybe I'll change it to "by". I don't want it to wrap.